### PR TITLE
Adding new swag opportunities 

### LINKED ISSUE #755: Xamarin Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Xamarin (Microsoft)

Q. Where have you seen the announcement (URL)?  
A. : https://devblogs.microsoft.com/xamarin/hacktoberfest-xamarin-community-toolkit/

Q. What kind of swag (clothing, stickers, ...)?  
A. : undisclosed

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### LINKED ISSUE #768: Zup IT Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Zup IT

Q. Where have you seen the announcement (URL)?  
A. : http://insights.zup.com.br/hacktoberfest

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing

Q. Have you received swag from this?  
A. : 

Q. How long did it take?  
A. : 

Q. Does it ship worldwide?  
A. : only Brazil

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by achaljhawar**
Fixes #755 
Fixes #768

### CODE REVIEW

**Comment by achaljhawar** [comment]
@aslafy-z pls me with this

File: data.json [hunk 1]
```diff
@@ -206,6 +206,15 @@
     "dateAdded": "2019-10-27T12:37:23.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "Earthly",
```
Comment (aslafy-z) [inline data.json:210]: As I said in #772, Earthly is handled by #729

File: data.json [hunk 2]
```diff
@@ -537,7 +546,7 @@
     "reference": "https://redislabs.com/try-redis-cloud-essentials-for-a-free-shirt/",
     "image": "https://i.imgur.com/pU8KfaE.png",
     "dateAdded": "2020-02-14T17:13:48.000Z",
-    "tags": ["clothing", "expired"]
+    "tags": ["clothing"]
```
Comment (aslafy-z) [inline data.json:549]: This opportunity is expired, please keep it as it is.

File: data.json [hunk 3]
```diff
@@ -718,5 +727,23 @@
     "image": "http://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2019/10/medium_25686-c9dd82896c9e66bb82600f256444abe6.png",
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
+  },
+  {
+    "name": "Xamarin by Microsoft",
```
Comment (aslafy-z) [inline data.json:732]: Please update the old  Xamarin opportunity instead of creating a new one.

File: data.json [hunk 4]
```diff
@@ -718,5 +727,23 @@
...
+    "description": "Make quality pull requests to issues that are 'approved item of work' t-shirt.",
+    "reference": "https://devblogs.microsoft.com/xamarin/hacktoberfest-xamarin-community-toolkit/",
+    "image": "https://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2020/10/image-1-1024x536.png",
+    "dateAdded": "2020-10-16T16:35:21.000Z",
+    "tags": ["hacktoberfest", "clothing"]
+  },
+  {
+    "name": "Zup IT",
+    "difficulty": "medium",
+    "description": "Get to PRs approved in the mentioned repos and the fill a form to get awesome t-shirt delivered at your home shipping is only their in Brazil.",
```
Comment (aslafy-z) [inline data.json:743]:
```suggestion
    "description": "Get two approved PRs in the mentioned repos and the fill a form to get an awesome t-shirt delivered at your home. (Ships only to Brazil)",
```

**Pull request closed without merging**